doc/source/templates/testing/cluster_envs/04_finetuning_llms_with_deepspeed.yaml

```diff
@@ -7,8 +7,8 @@ debian_packages:
 
 python:
   pip_packages: [
-    torch==2.0.0,
     torchvision==0.15.1,
+    peft,
```

Comment:
should we pin more versions here? so release tests don't go flaky so often?

Reply:
(peft, deepspeed, etc.)

Reply:
Yep, peft should be pinned. Thanks for noticing. I verified with 0.7.0.